### ISSUE TITLE
Critical vulnerabilities - Arbitrary code execution in lodash

### ISSUE THREAD

**Issue by ddbhagat** [lodash/lodash#5273]
NPM audit in my project started failing with below vulnarabilities errors. Please do the required

┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Critical      │ Arbitrary code execution in lodash                           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ lodash                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ No patch available                                           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ @loadable/server                                             │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ @loadable/server > lodash                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://github.com/advisories/GHSA-8p5q-j9m2-g8wr            │
└───────────────┴──────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Critical      │ Arbitrary code execution in lodash                           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ lodash                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ No patch available                                           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ redux-form                                                   │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ redux-form > lodash                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://github.com/advisories/GHSA-8p5q-j9m2-g8wr            │
└───────────────┴──────────────────────────────────────────────────────────────┘

**Comment by snabb** [comment]
Some moron created a false vulnerability report, see: https://github.com/lodash/lodash/issues/5261

**Comment by ddbhagat** [comment]
Yes We can see that this vunlnerability is withdrawn and under reanalysis 
https://github.com/advisories/GHSA-8p5q-j9m2-g8wr
https://nvd.nist.gov/vuln/detail/CVE-2021-41720

But most of us blocked until `npm audit` shows up this one !

**Comment by jdalton** [comment]
Dup of #5261.
